testdata/sqllogictests/functions/json_scan.slt

```diff
@@ -40,11 +40,6 @@ select * from ndjson_scan([]);
 
 # Glob patterns not supported on HTTP
 
-statement error Unexpected status code '404 Not Found'
-select * from ndjson_scan(
-  'https://raw.githubusercontent.com/GlareDB/glaredb/main/testdata/sqllogictests_datasources_common/data/*.ndjson'
```

Comment:
why did this error before and doesn't any more?

Reply:
So the previous error had both: status code and glob. Now the error message has changed...